### ISSUE TITLE
Some strings are missing for translation

### ISSUE THREAD

**Issue by ArionWT** [teeworlds/teeworlds-translation#71]
Hello,
There aren't some strings for translation on Transifex:

    Server
    body
    marking
    decoration
    hands
    feet
    eyes

([screenshots on Imgur](https://imgur.com/a/EM6UOK8))

If you add them to the .json file manually, translated strings works in the game (at least in version 0.7.2). Maybe this is some kind of bug in some convert script for loading on Transifex?

**Comment by Dune-jr** [comment]
Forwarded to https://github.com/teeworlds/teeworlds/issues/2027 since this is a code issue

**Comment by Dune-jr** [comment]
Fixed by https://github.com/teeworlds/teeworlds/pull/2029, updated Transifex

**Comment by ArionWT** [comment]
Seems other two strings don't work in the new menu:
`All` in `src\game\client\components\menus.cpp:2532`
`Favorites` in `src\game\client\components\menus.cpp:2530`
They are in .json file, but not applied ingame.
![_2019-02-10_00-06-49](https://user-images.githubusercontent.com/8038546/52526342-193cb100-2cc8-11e9-81d1-1b5bd7dc799f.png)

**Comment by Dune-jr** [comment]
Those are filters defined in `ui_settings.json`. Since filters can be customized by the user (but not really those), it would be difficult to localize them. 

@oy should we change that somehow and localize the default filter names, or leave it as is?

**Comment by oy** [comment]
@Dune-jr yeah, we could localize them.

**Comment by ArionWT** [comment]
Hello,
Missed one string on Transifex:
`Menu bakcground transparency`

And string №336 `You have to wait %1.f seconds to change identity.` on Transifex may be `You have to wait %1.0f seconds to change identity.` due this [commit](https://github.com/teeworlds/teeworlds/commit/da41a0973d191bc0c0df228006fac2c34bc6967d).

**Comment by Dune-jr** [comment]
Yeah, need to update Transifex for the menu background transparency. The second thing isn't a big deal, I can automate a fix.
